config/translate_test.go

```diff
@@ -421,8 +430,8 @@ func TestTranslateFromV1(t *testing.T) {
 
 	for i, test := range tests {
 		config := TranslateFromV1(test.in.config)
-		if !reflect.DeepEqual(test.out.config, config) {
-			t.Errorf("#%d: bad config: want %+v, got %+v", i, test.out.config, config)
+		if str := pretty.Compare(config, test.out.config); str != "" {
```

Comment:
I'd prefer depending on `testify/assert` and using `assert.Equal` over `pretty.Compare`.

assert also pretty-prints the diff, and provides better control over what 'equality' means.